recipes/r-ukbrapr/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set version = "0.2.9" %}
+{% set sha256 = "693c0ab85d21bea722d6f389898d75eddc3201e07bfb3b3a9e208290dd914d6e" %}
+
+package:
+  name: r-ukbrapr
+  version: {{ version }}
+
+source:
+  - url: https://github.com/lcpilling/ukbrapR/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+  script: R CMD INSTALL --build .
+  number: 0
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('r-ukbrapr', max_pin="x") }}
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  host:
+    - r-base
+    - r-dplyr >=1.1.0
+    - r-readr >=2.0.0
+    - r-stringr >=1.5.0
+    - r-purrr >=1.0.0
+    - r-lubridate >=1.9.0
+    - r-rlang >=1.0.0
+    - r-reticulate >=1.34.0
+    - r-arrow >=13.0.0
+    - r-sparklyr >=1.8.4
+    - r-cli >=3.6.1
+    - r-prettyunits >=1.0.0
+    - r-lifecycle >=1.0.0
+    - r-haven >=2.5.0
+  run:
+    - r-base
+    - r-dplyr >=1.1.0
+    - r-readr >=2.0.0
+    - r-stringr >=1.5.0
+    - r-purrr >=1.0.0
+    - r-lubridate >=1.9.0
+    - r-rlang >=1.0.0
+    - r-reticulate >=1.34.0
+    - r-arrow >=13.0.0
+    - r-sparklyr >=1.8.4
+    - r-cli >=3.6.1
+    - r-prettyunits >=1.0.0
+    - r-lifecycle >=1.0.0
+    - r-haven >=2.5.0
+
+test:
+  commands:
+     - $R -e 'library("ukbrapR")'
+
+about:
+  home: https://lcpilling.github.io/ukbrapR
```

Keep URLs together and use canonical `dev_url`.

```suggestion
  home: https://lcpilling.github.io/ukbrapR
  dev_url: https://github.com/lcpilling/ukbrapR
```